source/server/configuration_impl.cc

```diff
@@ -245,6 +231,25 @@ InitialImpl::InitialImpl(const envoy::config::bootstrap::v3::Bootstrap& bootstra
   }
 }
 
+void InitialImpl::initAccessLog(const envoy::config::bootstrap::v3::Bootstrap& bootstrap,
```

Comment:
Can we call this `initAdminAccessLog()`?

Reply:
ok